### ISSUE TITLE
Missing 'SPMASmoke.dds'

### ISSUE THREAD

**Issue by 100GPing100** [GreatEmerald/UT3Vehicles#177]
https://github.com/GreatEmerald/UT3Vehicles/blob/f8c45fe389fd190439c17e5f03ce1e6e207fa23a/Classes/UT3HellfireSPMAShellTrail.uc#L16
```UnrealScript
#exec texture import file=..\UT3Vehicles\Textures\SPMASmoke.dds
```

After binaries purge, with a clean repository clone (or a hard reset), I'm unable to compile due to the missing texture file. If this texture has already been imported into a texture package, then this line should be removed and the line where it is later used in that same class's `defaultproperties` should be changed accordingly.

**Comment by GreatEmerald** [comment]
Indeed. @HellDragon-HK @Unre-Alex Can you check whether that file is there somewhere already? If not, it needs to be put into `UT3SPMA.uax` (potentially also rename it to `UT3A_Vehicle_SPMA`).

**Comment by HellDragon-HK** [comment]
GreatEmerald commented 6 days ago

There's also Textures/SPMASmoke.dds, not sure what that's about. Probably Wormbo's, he likes not putting files into packages. But that won't work with the script, as it puts files directly into the needed folders. for now I'll keep it in the repository, but I'll remove it once that gets put into some package.
@HellDragon-HK
Contributor
HellDragon-HK commented 6 days ago •

SPMASmoke missing was something I was having as an error on compile when we first started this, I assumed you had to add it in.

There was also another error but unfortunately I can't remember what it was.

**Comment by GreatEmerald** [comment]
Right, you don't, I meant `UT3SPMATex.utx`.

**Comment by HellDragon-HK** [comment]
Ok I'll check the utx real quick to see if it's in there

**Comment by HellDragon-HK** [comment]
Nope it isn't
